### DISCUSSION TITLE
Unable to authenticate students.

### DISCUSSION THREAD

**Discussion by Jamkris** [community/community#138376] [GitHub Education]
### Hi everyone,

Hi everyone,
Unable to authenticate students.

Even if you get a certificate of enrollment in English, it will not be certified

<img width="321" alt="image" src="https://github.com/user-attachments/assets/b116e4c1-79bf-42ca-9793-fe55019be8a1">

But I'm not using a VPN.

**Comment by itisrude** [comment]
Hey Jamkris,

It looks like the issue might not be with using a VPN but with location services being blocked or not enabled. Even if you're not using a VPN, GitHub still needs to verify your campus location through your browser or operating system. Here are a few things you can try:

1. Enable Location Services: Make sure that your browser and OS have location services turned on. You might also need to grant specific permissions for GitHub to access your location.

2. Use a Different Browser or Device: If location services are enabled but still not working, try switching to a different browser or device to submit the application again. Sometimes certain browsers have stricter location security settings.

3. Verify School Information: Double-check the school information you provided. If the school has incorrect domains or campus locations, you may need to select "My selected school has incorrect or incomplete information" when submitting a support ticket.

If none of this works, reaching out to GitHub Education support through their [Student Developer Pack Application FAQ](https://github.com/orgs/community/discussions/111352) might be your best option.

Hope this helps!
